### PR TITLE
Hero search category

### PR DESCRIPTION
**Pull request by FayeCarter**

### CODE REVIEW

File: app/components/header/shared-header/account.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import BurgerSVG from "~/styles/svg/burger";
+import CloseSVG from "~/styles/svg/close";
```
Comment (charliearlie) [inline app/components/header/shared-header/account.tsx:2]: Can lucide-react for icons now

File: app/components/header/shared-header/shared-header.tsx [hunk 2]
```diff
@@ -1,76 +1,99 @@
...
+import type { Category } from "@prisma/client";
+import { Account } from "./account";
+import { Search } from "./search";
+import { Menu } from "./menu";
 
-import type { UserType as User } from "~/types";
+import { cn } from "~/util/utils";
+import { CategoryList } from "./category-list";
+
+import useScrollPosition from "../../../hooks/useScrollPosition";
```
Comment (charliearlie) [inline app/components/header/shared-header/shared-header.tsx:13]: pedantic but 

```typescript
import useScrollPosition from "~/hooks/useScrollPosition";
```

File: app/components/header/shared-header/account.tsx (hunk 1 above)
Comment (FayeCarter) [reply app/components/header/shared-header/account.tsx:2]: oof

File: app/components/header/shared-header/search.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import SearchSVG from "~/styles/svg/search";
```
Comment (charliearlie) [inline app/components/header/shared-header/search.tsx:1]: Can use Lucide React again

File: app/hooks/useScreenWidth.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
```
Comment (charliearlie) [inline app/hooks/useScreenWidth.ts:1]: You might hate my pedantry but might as well stick with kebab case naming convention for file names

**Comment by charliearlie** [comment]
Code wise, it's all good. I'll leave any visual comments on the preview link in this PR.

**Pull request merged**